Formula/kdoctor.rb

```diff
@@ -0,0 +1,25 @@
+class Kdoctor < Formula
+  desc "Environment diagnostics for Kotlin Multiplatform Mobile app development"
+  homepage "https://github.com/kotlin/kdoctor"
+  url "https://github.com/Kotlin/kdoctor/archive/refs/tags/v0.0.2.tar.gz"
+  sha256 "e95af8f76fb3a6240758af76b5528e57b41793065354c81bff2fbebe435f22df"
+  license "Apache-2.0"
+  head "https://github.com/Kotlin/kdoctor.git", branch: "master"
+
+  depends_on "openjdk@11"
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
+    mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+    build_task = "linkReleaseExecutableMacos#{mac_suffix}"
+    system "./gradlew", "clean", build_task
```

Comment:
Does this not work with the gradle formula?

Reply:
Yeah, that way we get all the fixes right away instead of after a wrapper update.